Consensus-Layer-Meetings/call_079.md

```diff
@@ -0,0 +1,234 @@
+It grew roughly with 7GB per day in the time of non-finalization.
+If we aim for 2 weeks of non-finalization max, that would be 98GB of additional space needed.
+I suspected the node to prune these states once we reached finalization.
+Is that something that clients do?
+Otherwise two or three periods of non-finalization would blow up most nodes
+- **From terence(prysmaticlabs) to Everyone 02:54 PM:**
+@zahary: you may be interested at this: https://github.com/ethereum/consensus-specs/pull/2802
+- **From Marius to Everyone 02:56 PM:**
+Been really helpful for us too, getting dos’d by teku did find a couple of weird races
+- **From James He to Everyone 02:57 PM:**
+keymanager-api standards providing a native way to onboard and offboard from a vc through API, prysm implementation of the 3 api endpoints in production, teku and lighthouse will have them in prod soon.
+Joaquim Vergès an android tech lead @twitch has been helping with the implementation on the 3 keymanager-apis on web3signer / developing a generic ui for these api endpoints providing a much needed community implementation of a gui for clients who implement these 3 endpoints.
+3 more keymanager-api endpoints were proposed for managing remote signer public keys.
+prysm currently developing web3signer support for end of january
+- **From protolambda to Everyone 03:05 PM:**
+the balances tree and participation flags tree cause the state diff to blow up more than desired
+the validator tree indeed doesn’t change much
+When will infura be serving light-client requests with merkle proof?
+- **From arnetheduck to Everyone 03:06 PM:**
+validator db is a bit more half of the state - it’s also the part that keeps growing, the rest stays constant-size more or less
+- **From protolambda to Everyone 03:07 PM:**
+constant-size for latest state, but not if you store every historical state. Then validators tree is one of the better parts (infrequent changes) afaik
+- **From Adrian Sutton to Everyone 03:08 PM:**
+Would be good to get light-client requests available soon - there’s just quite a bit going on with merge and key manager APIs. Do we have a standard API for the light client requests yet? I know Lodestar did some apis but not sure if they’ve been written up as a standard API yet.
+- **From protolambda to Everyone 03:09 PM:**
+Probably better to ask lodestar directly after call. Maybe with nimbus if they’re interested in consuming those proofs as well
+- **From arnetheduck to Everyone 03:10 PM:**
+linear history of blocks + one full state a day = 30gb
+- **From Trenton Van Epps to Everyone 03:12 PM:**
+https://ethresear.ch/t/whisk-a-practical-shuffle-based-ssle-protocol-for-ethereum/11763
+
+
+## Attendees
+- Trenton Van Epps
+- danny
+- Enrico Del Fante
+- Pooja Ranjan
+- Paul Hauner
+- lightclient
+- Saulius Grigaitis
+- Tim Beiko
+- Marius Van Der Wijden
+- ?
+- Ben Edgington
+- carlbeek
+- terence(prysmaticlabs)
+- Mikhail Kalinin
+- ?
+- Marek Moraczynski
+- ?
```

Comment:
Thanks for documenting. LGTM.
Please remove participants with "?"

Reply:
Thx for the review. It's done. I've also updated the participants list again so it's as complete as possible according to youtube screenshots